ci/bin/format.dart

```diff
@@ -822,7 +824,12 @@ class PythonFormatChecker extends FormatChecker {
       } else {
         error('Found ${incorrect.length} python file${plural ? 's' : ''}'
             ' which ${plural ? 'were' : 'was'} formatted incorrectly:');
-        incorrect.forEach(stderr.writeln);
+        stdout.writeln('To fix, run:');
```

Comment:
Should we recommend the `--fix` flag too? Perhaps:

```suggestion
        stdout.writeln('To fix, run `dart ci/bin/format.dart --fix` or:');
```

Reply:
Alternatively:

```suggestion
        stdout.writeln('To fix, run `et format` or:');
```